qualtran/bloqs/chemistry/thc/thc.ipynb

```diff
@@ -433,7 +433,7 @@
     "    print(f\"{k+':':20s} qualtran = {binned_counts[k]:5d} vs paper cost = {v:5d}.\")\n",
     "\n",
     "print(f\"Total cost = {sum(v for v in binned_counts.values())}\")\n",
-    "assert binned_counts['data_loading'] == 432"
+    "assert binned_counts['data_loading'] == 304"
```

Comment:
do you know why this changed?

Reply:
Because `find_optimal_log_block_size` is now called with the correct `target_bitsize` -- there was a bug where I was calling it with `selection_bitsize` instead so the block size was not optimal.